### ISSUE TITLE
Azurite does not launch in a docker image

### ISSUE THREAD

**Issue by Arash-Sabet** [Azure/Azurite#2365]
### Which service(blob, file, queue, table) does this issue concern?
All
### Which version of the Azurite was used?
Presumably the latest version
### Where do you get Azurite? (npm, DockerHub, NuGet, Visual Studio Code Extension)
npm
### What's the Node.js version?
Presumably the latest version
### What problem was encountered?
Azurite does not launch in a docker image as communication with it over port# 10000 shows connection failure
### Steps to reproduce the issue?
We followed the steps outlined in [this article ](https://programmingwithwolfgang.com/mastering-azure-devops-agents-in-kubernetes-guide/) to build a k8s Azure DevOps agent. We augmented [the docker yaml file](https://programmingwithwolfgang.com/mastering-azure-devops-agents-in-kubernetes-guide/#run-the-azure-devops-agent-locally) to include Azurite in the docker image to use its instance in our integration tests run by the build pipelines. The code snippet to include Azurite is per below:

```yaml
RUN sudo apt-get -y install nodejs
RUN sudo apt-get install -y nodejs npm
RUN sudo npm install -g azurite
RUN sudo npm fund
RUN sudo mkdir azurite
RUN sudo azurite --silent --location azuritefiles --debug azuritefiles\debug.log &
```
All build steps of our C# solution pass successfully using this k8s-backed build agent. Unfortunately, we realize the following error message produced by our Xunit integration tests when running them in the Azure DevOps pipeline: 

> Connection refused (127.0.0.1:10000)

The only architectural difference between our image and the one produced by the aforementioned article is that we are using Ubuntu 22.04 instead of Ubuntu 20.04.

### Have you found a mitigation/solution?
No

**Comment by Arash-Sabet** [comment]
@blueww The tests are run inside the docker image.

**Comment by blueww** [comment]
@Arash-Sabet 

I might still need some time to investigate this issue.
Would you please share the Azurite debug log which can help the investigation? 

Besides that, could you also try to start Azurite with parameter: --blobHost 0.0.0.0 --queueHost 0.0.0.0  --tableHost 0.0.0.0

**Comment by Arash-Sabet** [comment]
> @Arash-Sabet
> 
> I might still need some time to investigate this issue. Would you please share the Azurite debug log which can help the investigation?
> 
> Besides that, could you also try to start Azurite with parameter: --blobHost 0.0.0.0 --queueHost 0.0.0.0 --tableHost 0.0.0.0

I cannot access the log information as the docker image is a part of a k8s cluster. I will try the azurite per your suggestion a bit later on but bottom line the issue per my description should not happen.

**Comment by Arash-Sabet** [comment]
Running azurite on port# `0.0.0.0` worked.

**Comment by blueww** [comment]
Good to the issue is resolved by running Azurite with host as 0.0.0.0
I will close this issue.

Feel free to contact us again if need any further assistance on Azurite.

**Comment by Arash-Sabet** [comment]
Thanks for the help @blueww
